spec/features/work_show_spec.rb

```diff
@@ -1,4 +1,4 @@
-RSpec.describe "display a work as its owner" do
+RSpec.describe "work show view" do
   include Selectors::Dashboard
```

Comment:
Made a few modifications to test labels in this file because the output of the combined labels did not make sense.